triage/labels.rst

```diff
@@ -124,6 +124,8 @@ Other labels
 * :gh-label:`stale`: for issues/PRs that have been inactive for a while.
 * :gh-label:`sprint`: for easier filtering of issues/PRs being worked on
   during official sprints.
+* :gh-label:`infra`: for issues related to the infrastructure of the
+  project (e.g., GitHub Actions, dependabot, the buildbots, etc.).
```

Comment:
Let's avoid Latin: https://devguide.python.org/documentation/style-guide/#use-simple-language

```suggestion
  project (for example, GitHub Actions, dependabot, the buildbots).
```

And the list is mostly alphabetical, shall we sort it?

Reply:
[a9cd738](https://github.com/python/devguide/pull/1432/commits/a9cd738cde85809235c3e2e0de5197de4816cf26)